packages/cli/src/commands/translate/sync-keys/utils.ts

```diff
@@ -259,7 +259,10 @@ const traverseNode = async (
   await fs.writeFile(targetFilePath, '', { flag: 'w+' });
 
   if (isRoot) {
-    await fs.appendFile(targetFilePath, "import type { LocalePhrase } from '../../types.js';\n\n");
+    await fs.appendFile(
+      targetFilePath,
+      "import { type DeepPartial } from '@silverhand/essentials';\n\nimport type { LocalePhrase } from '../../types.js';\n\n"
```

Comment:
should we also use `DeepPartial` for the output type?

Reply:
Yes, otherwise the generated phrases will have a type error at this point. Otherwise we'll have to remove the existing `DeepPartial` usage from phrases. I'm good either way

Reply:
Offline discussed and this is OK